tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -247,7 +269,10 @@ def test_dhcp_relay_start_with_uplinks_down(ptfhost, dut_dhcp_relay_data, valida
        relay agent while the uplinks are still down. Then test whether the DHCP relay agent
        relays packets properly.
     """
-    testing_mode, duthost = testing_config
+    testing_mode, duthost, testbed_mode = testing_config
+
+    if testbed_mode == 'dual_testbed':
+        pytest.skip("skip the uplinks down testcase on dual tor testbeds")
```

Comment:
can we test the linkdown in the dual testbed mode.  I think we just need to switch the mux active after the the interface has been switched on. Feel free to do this in a subsequent PR.